tests/test_collinear_endpoint_cases.py

```diff
@@ -0,0 +1,46 @@
+def test_3():
+    # Test lots of collinear endpoint cases.
+    import random, math, pytest
+    from mt2 import mt2
+    random.seed(0) # If the test fails, we want to be able to repeat the test!
+    for i in range(10000):
+        m_visA = random.uniform(0,10)
```

Comment:
All binary operators should be surrounded by exactly one space on each side. There should be a space after a comma in eg lists of arguments.

Reply:
PEP 8 sec 2 defence: https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

Reply:
So basically you're using that as a "I can write however I like" get-out-of-jail-free card? Sorry, not happening :-)
I can fix the formatting though, and standardise it. Standards exist for a reason.